test/cookbooks/macos_test/recipes/xcode_from_url.rb

```diff
@@ -1,11 +1,11 @@
-if node['platform_version'] >= '10.14.4'
-  xcode 'installs 11.0' do
-    download_url node['xcode']['download_url']
-    version '11.0'
+if node['platform_version'] >= '10.15.6'
+  xcode 'installs 12.0' do
+    download_url ::File.read('/Users/vagrant/xcode')
```

Comment:
Why do we need to get the URL from a file?

Reply:
Kitchen lifecycle doesn't seem to have support for porting environment variables, and I didn't want to hardcode this. Previously, we set this via a node attribute in kitchen (`node['xcode']['download_url']`), and I wanted to be able to entirely delete the default attributes file.